### PR TITLE
Use hubot-routines package

### PR DESCRIPTION
**Pull request by desa85**

### CODE REVIEW

File: src/birthder.js [hunk 1]
```diff
@@ -12,11 +12,12 @@
...
 
 (function () {
-  const schedule = require('node-schedule')
+  const fs = require('fs')
   const moment = require('moment')
   const nFetch = require('node-fetch')
   const path = require('path')
-  const fs = require('fs')
+  const routines = require('hubot-routines')
+  const schedule = require('node-schedule')
```
Comment (eugulixes) [inline src/birthder.js:20]: Make a different PR and call it, for example, `Rearrange imports`

The commit message might look like the following.

```
Rearrange imports

Arrange them in alphabetical order.
```

**Comment by eugulixes** [comment]
I guess `hubot-routines` can be used without `.gitmodules`, so remove it.

**Comment by eugulixes** [comment]
Add yourself to `AUTHORS.md`.

**Review by polina-popova — APPROVED** [review]
